mkdocs.yml

```diff
@@ -86,6 +92,21 @@ extra:
       link: https://twitter.com/FraunhoferAISEC
       name: Follow Fraunhofer AISEC on Twitter.
 
+  consent:
+    title: Consent
+    description: >-
+      We use external services to enrich information presented on our website. This information is not essential 
+      for the operation of this website. You can opt-in, if you want to see additional information. Your choice 
+      will be saved in a cookie.
+    actions:
+      - reject
+      - accept
+      - manage
+    cookies:
+      github:
+        name: GitHub
+        checked: false
+
```

Comment:
This doesn't seem to work, the info from GitHub is loaded before the user selects any actions.

![image](https://user-images.githubusercontent.com/74261221/198011649-89abe620-ce70-4d6e-bd00-13f7d870b8b6.png)

Reply:
Never mind, was a problem on my side, everything works correctly :+1: 

![Screenshot_20221026_133416](https://user-images.githubusercontent.com/74261221/198015958-c32676be-14a6-4293-a21a-8f43fcff7f07.png)